src/object/object_primitive.c

```diff
@@ -17348,15 +17355,19 @@ mr_data_readmem_json (OR_BUF * buf, void *memptr, TP_DOMAIN * domain, int size)
       schema_str = db_get_string (&schema_raw);
     }
 
-  json->json_body = db_private_strdup (NULL, json_body_str);
   json->schema_raw = db_private_strdup (NULL, schema_str);
```

Comment:
Read memory should mirror write memory changes:
  1. Unpack directly from OR_BUF.
  2. No schema unpacking.